### PR TITLE
feature/container: Create Container component and add it to hui-pages

### PR DESCRIPTION
**Pull request by pixelfuture**
The Container component is ready

### CODE REVIEW

File: packages/container/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# hixme-ui checkbox
```
Comment (seesleestak) [inline packages/container/README.md]: Don't forget to update the README with Container documentation

File: packages/container/__tests__/index.test.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import { ThemeProvider } from '@hixme-ui/theme'
+import Container from '../src/index.js'
+
+test('Animate', () => {
```
Comment (seesleestak) [inline packages/container/__tests__/index.test.js]: Should probably say 'Container'. I've made a more generic version of this in a couple of other components (`/table/__tests__/index.test.js`, for example) that's a bit easier to just copy and paste.

File: packages/container/src/ContainerBase.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    if (props.flexBasis) output += `flex-basis: ${props.flexBasis};`
+    if (props.flexGrow) output += `flex-grow: ${props.flexGrow};`
+    if (props.flexShrink) output += `flex-shrink: ${props.flexShrink};`
+
+    return output
+  }
+  return ''
+}
+
+const getAnimate = ({ animate, heavyShadow }) => {
```
Comment (seesleestak) [inline packages/container/src/ContainerBase.js]: Super cool

File: packages/container/src/ContainerBase.js [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+    if (textCenter) return 'center'
+    if (textRight) return 'right'
+    return 'inherit'
+  }};
+  background: ${(props) => {
+    if (props.background) return props.background
+    return getBackgroundColor(props, 'none')
+  }};
+  ${getFlexProps}
+  ${({ width }) => width && `width: ${width};`}
```
Comment (seesleestak) [inline packages/container/src/ContainerBase.js]: Really dig this syntax

File: packages/table/package.json [hunk 5]
```diff
@@ -42,7 +42,6 @@
     "David Cotelessa"
   ],
   "devDependencies": {
-    "babel": "^6.23.0",
```
Comment (seesleestak) [inline packages/table/package.json]: Thanks

File: packages/container/README.md (hunk 1 above)
Comment (billyxs) [reply packages/container/README.md]: Lots of documentation in your future lol.  Let me know if you want an assist on adding tests or documentation

**Comment by pixelfuture** [comment]
Documentation and tests completed. Please send me more feedback.

File: packages/container/README.md [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+| flexGrow            | string      | Apply CSS flex-grow properties       |
+| flexShrink          | string      | Apply CSS flex-shrink properties     |
+
+
+## Other
+
+| Name            | Type        | Description                           |
+| --------------- | ----------- | ------------------------------------- |
+| withOutline     | bool        | Add 1px solid red outline             |
+| noPadding       | bool        | Remove default padding from container |
```
Comment (billyxs) [inline packages/container/README.md]: If I want to find this in the padding section?

Comment (pixelfuture) [reply packages/container/README.md]: Great idea. Just pushed this update.

**Pull request merged**